inspirehep/modules/disambiguation/receivers.py

```diff
@@ -92,7 +92,7 @@ def _needs_beard_reprocessing(authors_before, authors_after):
         return True
 
 
-@after_record_insert.connect
+# @after_record_insert.connect
```

Comment:
That is not constructive and not generous. The whole module is not broken. It might have certain bad design decisions WRT handling record updates which do not justify removing the whole PR.

I am fine with detaching the signals as proposed in this PR, though.

We need to have disambiguation working for in 2 milestones. (i.e. after Invenio 3, and search)